test/Microsoft.Framework.PackageManager.FunctionalTests/DnuTestUtils.cs

```diff
@@ -11,11 +11,56 @@ namespace Microsoft.Framework.PackageManager
 {
     public static class DnuTestUtils
     {
-        public static int ExecDnu(string runtimeHomePath, string subcommand, string arguments,
-            IDictionary<string, string> environment = null, string workingDir = null)
+        public static int ExecDnu(string runtimeHomePath, 
+                                  string subcommand, 
+                                  string arguments, 
+                                  out string stdOut, 
+                                  out string stdErr,
+                                  IDictionary<string, string> environment = null, 
+                                  string workingDir = null)
         {
-            var runtimeRoot = Directory.EnumerateDirectories(Path.Combine(runtimeHomePath, "runtimes"), Constants.RuntimeNamePrefix + "*").First();
             string program, commandLine;
+            BuildProcessInformation(runtimeHomePath, subcommand, arguments, out program, out commandLine);
+
+            var exitCode = TestUtils.Exec(program, commandLine, out stdOut, out stdErr, environment, workingDir);
+
+            return exitCode;
+        }
+
+
+        public static int ExecDnu(string runtimeHomePath,
```

Comment:
Good idea. But the interface is used by quite a few other tests. I'm not going to change it's signature now. I'll move for another separate PR to do this.